lib/usd/translators/jointWriter.cpp

```diff
@@ -222,16 +221,16 @@ static bool _FindDagPoseMembers(
     indices->resize(numDagPaths);
 
     std::vector<uint8_t> visitedIndices(numDagPaths, 0);
-    for (unsigned int i = 0; i < membersPlug.numElements(); ++i) {
+    for (unsigned int i = 0; i < membersPlug.numConnectedElements(); ++i) {
```

Comment:
I don't mind so much because I think this makes sense (this connection needs to exist anyway to be meaningful so the change will work), but what catches my eye is the reasoning; your comment seems suspicious that somehow `connectedTo` created a new element. Might be worth trying using `evaluateNumElements` to see if this is somehow an evaluation caching issue and if so worth reporting as a separate Maya bug.

Reply:
Yeah, the thing where it was somehow creating an element was odd.  Unfortunately, I didn't have time to fully investigate at the time, and I don't know if I'd be able to replicate now.  I guess I also didn't feel too motivated too, since I feel like using `.numConnectedElements()` is conceptually better anyway.
If you want, I can edit the commit message, and remove the bit about the crash / element being created, or throw a bunch more caveats around it.

Reply:
Not that big of a deal; if it's not something that you cannot reliably reproduce already, this code is fine as is (IMHO).

Reply:
To be clear - does the thumbs up mean you're ok with this as is? Or that you'd like me to edit the commit message?